networking/advertise-service-ips.md

```diff
@@ -142,6 +144,82 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
    For help see, [BGP configuration resource]({{ site.baseurl }}/reference/resources/bgpconfig).
 
+#### Advertise service load balancer IP addresses
+
+The following steps will configure {{site.prodname}} to advertise Service `status.LoadBalancer.Ingress.IP` addresses.
+
+1. Identify the IP ranges to be used for Service LoadBalancer address allocation.
+
+1. Check to see if you have a default BGPConfiguration.
+
+   ```bash
+   calicoctl get bgpconfig default
+   ```
+
+1. Based on above results, update or create a BGPConfiguration.
+
+   **Update default BGPConfiguration**
+   Patch the BGPConfiguration using the following command, adding your own service load balancer IP CIDRs:
+
+   ```bash
+   calicoctl patch BGPConfig default --patch '{"spec": {"serviceLoadBalancer": [{"cidr": "x.x.x.x/16"}]}}'
+   ```
+
+   **Create default BGPConfiguration**
+   Use the following sample command to create a default BGPConfiguration. Add your CIDR blocks for load balancer IPs to be advertised in the `serviceLoadBalancerIPs` field.
+
+   ```bash
+   calicoctl create -f - <<EOF
+   apiVersion: projectcalico.org/v3
+   kind: BGPConfiguration
+   metadata:
+     name: default
+   spec:
+     serviceLoadBalancerIPs:
+     - cidr: x.x.x.x/16
+   EOF
+   ```
+
+   For help see, [BGP configuration resource]({{ site.baseurl }}/reference/resources/bgpconfig).
+
+Service LoadBalancer address allocation is outside the current scope of {{site.prodname}}, but can be implemented with an external controller.
+You can build your own, or use a third-party implementation like the MetalLB project.
+
+To install the MetalLB controller for allocating addresses, perform the following steps.
+
+1. Follow [the MetalLB documentation](https://metallb.universe.tf/installation/#installation-by-manifest) to install the `metallb-system/controller` resources. However, do not install the `metallb-system/speaker` component.
```

Comment:
This is a sentence with a whole lot behind it.  "do not install the `metallb-system/speaker`" - the instructions at the URL given don't actually say how to exclude the `metallb-system/speaker` component.  They just provide a single giant manifest that installs all the resources needed for both speaker and controller components.

Do we want to provide any guidance on that or are we leaving that as an exercise for the user?

Reply:
My hope was to leave it as an exercise for the reader. Yeah, it's not great, because it requires them to read and understand the MetalLB manifest to an extent. But selfishly, I don't want to be in the business of supporting our own fork of the MetalLB manifests. Open to other ideas, though.

Reply:
It seems ok to punt to user, but we might want to say **why** they should not install that component. If bad things happen or nothing works, it is added incentive to make sure they figure it out rather than ignore...which could be tempting.